docs/source/articles/storefront-search-features.html.md

```diff
@@ -452,7 +450,10 @@ You will see the fields referenced this way within Elasticsearch request bodies
 
 The namespaces (e.g. `active` and `content`) perform two primary functions.
 The `active` namespace is a special case which groups fields representing the activeness of products.
-This field allows for more accurate previewing of releases in the Storefront (inactive products are excluded from results).
+These fields allows for more accurate previewing of releases in the Storefront (inactive products are excluded from results).
+Before Workarea 3.5, these fields keyed off of releases (storing a per-release active value).
+Since Workarea 3.5, they key off of segments.
```

Comment:
This won't really be true in the final implementation, I changed it to simplify: https://github.com/workarea-commerce/workarea/blob/make-activeness-per-segment/core/app/models/workarea/search/storefront.rb#L40-L51